File: src/main/scala/com/bombproj/model/DataTableIndexes.java
package com.bombproj.model;

import com.queryflow.annotation.Table;
import lombok.Data;

@Data
@Table("datatable_indexes")
public class DataTableIndexes {

    private Long id;
    private String type;
    private String method;
    private String sort;
    private String dbType;
    private Integer sequence;

}

File: src/main/scala/com/bombproj/model/DataTableIndexes.scala
package com.bombproj.model

import com.queryflow.annotation.Table

import scala.beans.BeanProperty

@Table("datatable_indexes")
case class DataTableIndexes() {

    @BeanProperty var id: Long = _
    @BeanProperty var `type`: String = _
    @BeanProperty var method: String = _
    @BeanProperty var sort: String = _
    @BeanProperty var dbType: String = _
    @BeanProperty var sequence: Int = _

}
